python/brayns/plugins/circuit/circuit_color_method.py

```diff
@@ -24,6 +24,7 @@
 class CircuitColorMethod:
     """Available coloring methods for circuits."""
 
+    ELEMENT_ID: ClassVar[str] = 'element id'
```

Comment:
I would just call that ID as the previous function was color_circuit_by_id and the class name is already quite long...

But just personal preference, if you prefer it like that it is fine also.

Reply:
Renamed to id